packages/api/src/entity/library_item_preview.ts

```diff
@@ -0,0 +1,43 @@
+import {
```

Comment:
I think we could leave this one out for now. Since we are still kind of working out the social bits.

Reply:
I also think this might cause some RLS issues, since user B wouldn't be able to access the library_items of user A.